### PR TITLE
Stroke Replacement

### PR DESCRIPTION
**Pull request by iAkashPattnaik**
The Border / Stroke Of Whole SVG Makes The Card Look **Ugly** !

### CODE REVIEW

**Comment by tuhinpal** [comment]
changing card color to none is not an option yo can again send another pr that has another border color. when someone use that the border color separate it from background

**Comment by iAkashPattnaik** [comment]
ok then
sending another in 10 mins

**Comment by iAkashPattnaik** [comment]
added new query / customization
`stroke=none` or `stroke=fff`

**Comment by iAkashPattnaik** [comment]
what?

This Was Completely A New Customization

File: template/card.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ async function main(name, pic, repo, stars, commits, forks, tis, cis, text_color
             }
         }
     </style>
-    <rect x="0.5" y="0.5" rx="4.5" height="99%" width="494" fill="#${card_color}" stroke="none" stroke-opacity="1" />
+    <rect x="0.5" y="0.5" rx="4.5" height="99%" width="494" fill="#${card_color}" stroke="{stroke}" stroke-opacity="1" />
```
Comment (tuhinpal) [inline template/card.js:63]: This won't work the line should be like this 
stroke="${stroke}"

File: api/index.js [hunk 2]
```diff
@@ -14,6 +14,12 @@ module.exports = (req, res) => {
         if (cc == undefined) var cc = 'fff';
         if (tc == undefined) var tc = '000';
         if (ic == undefined) var ic = 'FF0000';
+        var stroke = req.query.stroke;
+        if (stroke == undefined || stroke == 'none') {
+            var stroke = 'none'
+        } else {
+            var stroke = `#`
```
Comment (tuhinpal) [inline api/index.js]: where is color code in this line?

**Comment by tuhinpal** [comment]
Due to many error in codes I am closing this PR, thanks for Contributing

**Comment by iAkashPattnaik** [comment]
i forgot theese
sorry for wrong pr
and thanks for informing

**Pull request closed without merging**